### DISCUSSION TITLE
Income and Expense Accounts Missing in Source/Destination Fields for Transactions in Firefly III

### DISCUSSION THREAD

**Discussion by V373z** [firefly-iii/firefly-iii#9402] [Firefly III] [answered]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am trying to set up Firefly III, and I’ve listed all spending categories under “Expense Accounts.” When I attempt to make an expense transaction, ideally, one of these expense accounts should be selectable as the “Destination Account.” However, only asset accounts are displayed as options for both the “Source Account” and “Destination Account” fields. Similarly, when recording income transactions, only asset accounts are listed, with income or expense accounts not showing up at all.

Source
![Image](https://github.com/user-attachments/assets/f4a41fd6-92f4-4557-83ef-e742f43a2ff4)

Destination
![Image](https://github.com/user-attachments/assets/b27f5af7-6c18-4433-8183-7e8467bf7fbc)

In Recurring Transactions, however, the expenditure accounts are displayed
![Image](https://github.com/user-attachments/assets/11526c39-99f2-4e0b-8a23-36c13f96be94)

### Debug information

??

**Comment by JC5** [comment, marked as answer]
Hi, thanks for opening an issue. This is not a bug.

The fields you open are autocomplete fields and will show the correct accounts once you start typing input. You are free to use the suggested inputs, or create new expense / revenue accounts.

**Reply by V373z** [reply]
Hi, thanks for your answer! That makes sense now, of course. I thought I could only select what was suggested in the dropdown list.
But if a dropdown list is already displayed, then as a user I would have expected it to be the same as it is displayed for Recurring Transactions under Destination Account and also that you can see that you can enter a new Destination Account. Then it will be the same on both screens. Just my suggestion. Thank you again, I'm really happy now!
